docs/sections/papers/ultrafeedback.md

```diff
@@ -1,3 +1,149 @@
 ## UltraFeedback: Boosting Language Models with High-quality Feedback
 
-...
+UltraFeedback is a large-scale, fine-grained, diverse preference dataset, used for training powerful reward models and critic models.
+
+UltraFeedback collects about 64k prompts from diverse resources (including UltraChat, ShareGPT, Evol-Instruct, TruthfulQA, FalseQA, and FLAN), then they use these prompts to query multiple LLMs (commercial models, Llama models ranging 7B to 70B, and non-Llama models) and generate four different responses for each prompt, resulting in a total of 256k samples i.e. the UltraFeedback will rate four responses on every OpenAI request.
```

Comment:
a pool of multiple LLMs?

Reply:
That's retrieved from their dataset card at https://huggingface.co/datasets/openbmb/UltraFeedback